test/unit/ui/handler/drag_rotate.test.js

```diff
@@ -273,6 +273,7 @@ test('DragRotateHandler ensures that map.isMoving() returns true during drag', (
 
     simulate.mousedown(map.getCanvas(), {buttons: 2, button: 2});
     simulate.mousemove(map.getCanvas(), {buttons: 2, clientX: 10, clientY: 10});
+    map._renderTaskQueue.run();
```

Comment:
This particular test is now failing from CI, is this call actually processing the events correctly?

Reply:
I was missing another one of these calls after the next event.

In old releases `map.isMoving()` would return true immediately after a `mousemove` event that causes a rotation. Now `map.isMoving()` only returns true on the next frame when the move events have fired and the camera events have changed. Does this seem ok to you?

Reply:
I would not think that having a one frame delay difference for this type of API as being critical. As long as we still cover the following requirements:
>Returns true if the map is panning, zooming, rotating, or pitching due to a camera animation or user gesture.